applications/NXmx.nxdl.xml

```diff
@@ -605,6 +605,11 @@
 							of dimensions **np** by **m** (slow to fast) with the
 							relative weights in incident_wavelength_weight as a 2D
 							array.
+
+						        Note, variants are a good way to represent several of
+							these usecases in a single dataset, e.g. if a calibrated,
```

Comment:
two words: `use cases`

Reply:
Also, the indentation here is not consistent with the previous lines.  This is the reason for the failure on [travis-ci](https://travis-ci.org/github/nexusformat/definitions/jobs/675042570#L967-L986).  Copy the leading white space from the previous line verbatim and use as the indentation for each of the following lines in this part.